[7983] Change apostrophes from straight to curly style (#5043)
DFE-Digital/register-trainee-teachers: docs/healthcheck_and_ping_endpoints.md

#### Failures caused by stale jobs

> This check will fail if there are stale jobs in a queue that doesn’t have a

Reviewer: ```suggestion
This check will fail if there are stale jobs in a queue that does not have a
```

end
```

> If you see jobs in a queue that shouldn’t be running any more (e.g. we’ve retired

Reviewer: ```suggestion
If you see jobs in a queue that should not be running any more (for example, we’ve retired
```

> `find_sync` and shouldn’t see any jobs for it) then you’ll need to remove any

Reviewer: ```suggestion
`find_sync` and should not see any jobs for it) then you’ll need to remove any
```
